classes/01class/aws101/README.md

```diff
@@ -32,6 +32,8 @@
     - Automation
         - Automation is an important factor to be able to recreate environments faster and without human errors
 
+![IMAGE](assets/cloud.png)
```

Comment:
Can you add some comments around this image? It's clear for me what it means, but for someone whithout all the information, it might not make a lot of sense.